doc/source/whatsnew/v0.17.0.txt

```diff
@@ -226,11 +226,13 @@ or it can return False if broadcasting can not be done:
 
 Other API Changes
 ^^^^^^^^^^^^^^^^^
+
 - Enable writing Excel files in :ref:`memory <_io.excel_writing_buffer>` using StringIO/BytesIO (:issue:`7074`)
 - Enable serialization of lists and dicts to strings in ExcelWriter (:issue:`8188`)
 - Allow passing `kwargs` to the interpolation methods (:issue:`10378`).
 - Serialize metadata properties of subclasses of pandas objects (:issue:`10553`).
-
+- Boolean comparisons of a ``Series`` vs ``None`` will now be equivalent to comparing with ``np.nan``, rather than raise ``TypeError``, xref (:issue:`1079`).
```

Comment:
```
In [1]: s = Series(range(3))

In [2]: s==None
TypeError: Could not compare <type 'NoneType'> type with Series
```

Its actually a really tiny edge case. The point of this PR was really to remove deprecation warnings  (from numpy) in the builds.